### PR TITLE
TE-7723 Fix broken master for spryker-sdk/spryk

### PR DESCRIPTION
**Pull request by AsonUnique**
### Overview 
- https://spryker.atlassian.net/browse/TE-7723

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "require-dev": {
     "codeception/codeception": "^4.0.0",
     "phpstan/phpstan": "^0.12.0",
+    "phpunit/phpunit": "^6.0.0 || ^7.0.0 || ^8.0.0",
```
Comment (dereuromark) [inline composer.json:22]: Are you sure we need also 6?

Comment (AsonUnique) [reply composer.json:22]: Our test will work with phpunit 6 and 7 (but not with 5), if you think we don't need it, we can leave only phpunit 8.

Comment (dereuromark) [reply composer.json:22]: Is there any travis run that was able to install it?
As I said, the PHP requirement above is php73+, and I doubt that will install 6 or 7. Please double check.

Comment (AsonUnique) [reply composer.json:22]: `PREFER_LOWEST` travis build installs phpunit 6.0.0, I also tried 7.0.0 locally and it works well.

Comment (dereuromark) [reply composer.json:22]: Then restrict it to 7.
There is no need to keep dev deps this low, better to keep up to date here.

**Review by gechetspr — APPROVED** [review]
Everything looks good, except phpunit version

**Pull request merged**